clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -181,7 +181,15 @@ func (p *profileMeasurement) gatherProfile(c clientset.Interface) ([]measurement
 		return nil, fmt.Errorf("profile gathering failed finding component port: %v", err)
 	}
 	profileProtocol := getProtocolForComponent(p.config.componentName)
-	getCommand := fmt.Sprintf("curl -s -k %slocalhost:%v/debug/pprof/%s", profileProtocol, profilePort, p.config.kind)
+
+	var getCommand string
```

Comment:
Can we extract this logic (constructing getCommand) to a separate method?

Reply:
Ok, I'll do that

Reply:
Done in https://github.com/kubernetes/perf-tests/pull/1136/commits/b6e71483ad60e548c065d1ed905ba87e101e5809